src/event.js

```diff
@@ -573,7 +574,7 @@ jQuery.Event.prototype = {
 
 		this.isPropagationStopped = returnTrue;
 
-		if ( e ) {
+		if ( e && this.executeOriginalPropagation ) {
```

Comment:
I can go either way though, the only requirement i would want to leave in is to save `originalEvent` property